lib/credential/data/dtos/claim_proofs/claim_proof_dto.dart

```diff
@@ -6,44 +6,44 @@ import 'claim_proof_sm_dto.dart';
 
 part 'claim_proof_dto.g.dart';
 
-@JsonEnum()
-enum ClaimProofType {
-  @JsonValue("BJJSignature2021")
-  bjj,
-  @JsonValue("Iden3SparseMerkleProof")
-  sparseMerkle
-}
-
```

Comment:
You should add "Iden3SparseMerkleTreeProof" as another enum and add it to the "case ...:".

Reply:
no cause we need to support any proof type, not only those 2 or 3 ones

Reply:
this was making the app fail, and in case of this happening in the future, claim should be able to be fetched

Reply:
@Plumillon Are we aligned? Asking since this is a hot fix that needs to be out there asap and to unblock moving forward.